packages/angular-display/src/lib/display.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import { IsDisplayDirective } from './directive/is-display.directive';
+
+@NgModule({
+  declarations: [IsDisplayDirective],
+  exports: [IsDisplayDirective],
+})
+export class DisplayModule {}
```

Comment:
you should add a `forRoot` static method here, so that users can define their breakpoints.

And as we talked about it would also be cool if we could expose the configured breakpoints as CSS custom properties for our users as well: https://css-tricks.com/updating-a-css-variable-with-javascript/

Reply:
but it's not exported in the `index.ts` and I also don't think we should.

We should stick to the `forRoot` patter which they describe in the [Angular-Docs](https://angular.io/guide/singleton-services#how-forroot-works) and which is used for configuring the [Angular-Router](https://angular.io/api/router/RouterModule) and also used to configure Services used by the [ngx-translate](https://github.com/ngx-translate/core#1-import-the-translatemodule) lib.

Reply:
They use that because they have complex configurations. We don't.

Reply:
Added the missing export.